### ISSUE TITLE
the issue about dataloader ?

### ISSUE THREAD

**Issue by JiaxingChai** [GewelsJI/VPS#35]
for idx, (img, label) in enumerate(zip(img_li, label_li)):
      if idx == 0:
          IMG = torch.zeros(len(img_li), *(img.shape))
          LABEL = torch.zeros(len(img_li) - 1, *(label.shape))
          IMG[idx, :, :, :] = img
      else:
          IMG[idx, :, :, :] = img
          LABEL[idx - 1, :, :, :] = label

so, why LABEL has less frame than IMG ? i 've been confused about it ?
